Source/Scene/Cesium3DTile.js

```diff
@@ -120,6 +120,23 @@ define([
         this.children = [];
 
         /**
+         * Descendant tiles that need to be visible before this tile can refine. For example, if
+         * a child is empty (such as for accelerating culling), its descendants with content must
+         * be loaded first. If a tiles's children all have content, this is left undefined.
+         *
+         * @type {Array}
+         * @readonly
+         */
+        this.descendantsWithContent = undefined;
```

Comment:
Perhaps mention in the comment that this is computed at runtime, not loaded from tileset.json.

Also, as we discussed before, but now that you have more context, would it be useful for tileset.json to have any metadata for this, e.g., number of descendants with content?